File: minio/reader.js
const Minio = require('minio')

// Initialize MinIO client
const client = new Minio.Client({
  endPoint: '135.181.91.126',     //ip of the minio server
  port: 9000,
  useSSL: false,
  accessKey: 'uKrvnXQDDNPJ0Q5v3eyI',  //access key of minio
  secretKey: 'f2JIQ9HGZ8cTBDDzByJ1SZpuUgaHHQ1mgRrANABy' //secret key of minio
})

// Specify bucket name and file name
const bucketName = 'my-bucket'
const fileName = 'test.txt'

// Get object data and print content
client.getObject(bucketName, fileName, (err, dataStream) => {
  if (err) {
    return console.log(`MinIO error occurred: ${err}`)
  }
  
  let content = ''
  dataStream.on('data', (chunk) => {
    content += chunk
  })

  dataStream.on('end', () => {
    const jsonData = JSON.parse(content)   //convert to json
    const name = jsonData.email            //json key 
    console.log('Content of key:')
    console.log(`Value of key: ${name}`)
  })
})

File: minio/reader.py
from minio import Minio
from minio.error import S3Error

# Initialize MinIO client
client = Minio(
    "135.181.91.126:9000",
    access_key="uKrvnXQDDNPJ0Q5v3eyI",
    secret_key="f2JIQ9HGZ8cTBDDzByJ1SZpuUgaHHQ1mgRrANABy",
    secure=False  # Change to True if MinIO server uses HTTPS
)

# Specify bucket name and file name
bucket_name = "my-bucket"
file_name = "test.txt"

try:
    # Get object data
    data = client.get_object(bucket_name, file_name)
    
    # Read and print object content
    content = data.read().decode('utf-8')
    print("Content of test.txt:")
    print(content)
    
except S3Error as err:
    print(f"MinIO error occurred: {err}")
